usaspending_api/api_contracts/contracts/v2/references/filter_tree/tas.md

```diff
@@ -0,0 +1,1035 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# TAS
+
+These endpoints are used to power USAspending.gov's TAS search component on the advanced search page.
+The response is a forest of filter search nodes, which despite having a unified structure represent different
+database fields based on depth in the tree.
+
+## Toptier Search [GET /api/v2/references/filter_tree/tas/{?depth}]
+
+Returns a list of toptier agencies that have at least one TAS affiliated with them
++ Request A request with a contract id
+    + Parameters
+        + `depth`: `0` (optional, enum[number]) How many levels deep the search will populate each tree. 
+            + Members
+                    + `0`
+                    + `1`
+                    + `2`
+        0 will return only agencies, 1 will return agencies and any federal accounts under them, and so on.
+    + Schema
+    
+            {
+            "$schema": "http://json-schema.org/draft-04/schema#",
+            "type": "string"
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, array[TASFilterTreeNode], fixed-type)
+    + Body
+
+            {
+            "results": [
+            {
+            "id": "012",
+            "ancestors": [],
+            "description": "Department of Agriculture",
+            "count": 139,
+            "children": null
+            }
+            ]
+            }
+
+
+## Search by Agency [GET /api/v2/references/filter_tree/tas/{agency}/{?depth}]
+
+Returns a list of federal accounts associated with the specified agency
++ Request A request with an agency id
+    + Parameters
+        + `agency`: `020` (required, string) 
+        + `depth`: `0` (optional, enum[number]) How many levels deep the search will populate each tree. 
+            + Members
+                    + `0`
+                    + `1`
+                    + `2`        
+        0 will return only federal accounts, and 1 will return federal accounts and any TAS under them
+    + Schema
+    
+            {
+            "$schema": "http://json-schema.org/draft-04/schema#",
+            "type": "string"
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `results` (required, array[TASFilterTreeNode], fixed-type)
+    + Body
+
+            {
+            "results": [
+            {
+            "id": "012-8226",
```

Comment:
Is there a benefit having 800+ lines of a response? Truncating to a few result objects should be sufficient to document the response and for a mock server